### DISCUSSION TITLE
Column selection preferences in settings (instead of cookies)

### DISCUSSION THREAD

**Discussion by NicoLeOca** [pymedusa/Medusa#3464] [Ideas & Feature requests]
Hi,

Would it be possible to store the preferences of the column selection in the settings?
Cookies are getting erased more and more by different softwares and it's a pita to set it back up.
exceptions in the many softwares are not a prefered solution at all.
![image](https://user-images.githubusercontent.com/9487685/33977367-d761ac74-e09a-11e7-8737-f62623246ae5.png)

**Comment by Rouzax** [comment]
Yes please.

**Comment by p0psicles** [comment]
I guess we can do that. But then it would be one configuration for all shows.

**Comment by fernandog** [comment]
imo i like the way it is. So I can have one configuration for mobile, one for my huge screen desktop and one for my small tablet

**Comment by gmusgrave** [comment]
Can this at least be done for the Home Page? 

The preferred column sorting (for me this is Next Episode ascending), and column selections (from the drop-down list) are not sticky, and need to be constantly reset. In my opinion, it would be a great improvement if these were stored somewhere.

Also, unless I'm missing something, these don't seem to be being stored in cookies. At least I can't find a cookie. Is it written immediately after you change the settings? If so, where is it. There doesn't seem to be one associated with localhost.

**Comment by fernandog** [comment]
@OmgImAlexis  can you check please?

**Comment by gmusgrave** [comment]
What is the status?

Has sticky selection for sorting and columns been implemented for the Home Page?

This is a minor annoyance, but an annoyance nonetheless.
